NEWS

```diff
@@ -1,6 +1,9 @@
 Ticket numbers in this file can be looked up by visiting
 http://twistedmatrix.com/trac/ticket/<number>
 
+.. towncrier release notes start
```

Comment:
Is this intended to be parsed as ReST? If so, would it make sense to rename this file `NEWS.rst`?